src/osx/carbon/font.cpp

```diff
@@ -366,6 +393,11 @@ void wxFontRefData::MacFindFont()
                         m_ctFont.reset(fontWithTraits);
                 }
             }
+            CFDictionarySetValue(dict, kCTFontAttributeName, m_ctFont.get() );
+            CFDictionarySetValue(dict, kCTForegroundColorFromContextAttributeName, kCFBooleanTrue);
+            
+            fontcache[ std::wstring(lookupnameWithSize.wc_str()) ] = m_ctFont;
```

Comment:
`s.ToStdWstring()` is a shorter and more clear way of writing `std::wstring(s.wc_str())` (this applies here and in a few other places).